optimizer.py

```diff
@@ -65,3 +67,25 @@ def fitness(solution, solution_idx):
     print('Found solution with cost: ', plan.cost(), ' after ', fitness.cost_recalc_counter,
           ' cost recalculations.')
     return plan
+
+
+def bees_alg_opt(plan):
```

Comment:
We probably should document implementations somehow. Please decide on the best approach -- either in code or a separate set of readmes.